template_overlay_typescript_components/lumin/package.json

```diff
@@ -30,7 +29,7 @@
   "dependencies": {
     "eslint-plugin-react": "^7.15.1",
     "magic-script-components": "^2.0.2",
-    "magic-script-components-lumin": "^1.0.13",
+    "magic-script-components-lumin": "^1.0.11",
```

Comment:
this shouldn't be changed to 11 but stay as 13

Reply:
Actually, it needs to be `^1.0.14` :)